.gitignore

```diff
@@ -114,3 +114,7 @@ dist
 # End of https://www.toptal.com/developers/gitignore/api/node
 
 .DS_store
+
+hubspot.config.yml
+/tmp
+/temp
```

Comment:
What was creating a `temp` directory?

Reply:
Me 😂  Do you think we should remove this?

Reply:
Yeah, likely not a common pattern ;).